apps/storefront/src/utils/loginInfo.ts

```diff
@@ -112,7 +112,7 @@ export const getloginTokenInfo = () => {
 export const loginInfo = async () => {
   const loginTokenInfo = getloginTokenInfo();
   const {
-    data: { token },
+    storeFrontToken: { token },
```

Comment:
As with previous changes like this, we might want to absorb this destructuring into the `getBCGraphqlToken` method:

```tsx
const token = await getBCGraphqlToken(loginTokenInfo);
```